code/modules/power/cell.dm

```diff
@@ -293,9 +299,6 @@
 	rating = 6
 	chargerate = 30000
 
-/obj/item/stock_parts/cell/infinite/use()
-	return 1
```

Comment:
Why did you remove this?

Reply:
Quite frankly I don't remember, most likely an accident, fixing now.